app/src/common/constants/permissions.js

```diff
@@ -103,6 +104,7 @@ export const PERMISSIONS_MAP = {
     [ACTIONS.WORK_WITH_TESTS]: true,
     [ACTIONS.SEE_EMAIL_MEMBERS]: true,
     [ACTIONS.SEE_ROW_ACTION_MENU]: true,
+    [ACTIONS.MANAGE_TEST_ITEMS_ACTIONS]: true,
```

Comment:
Can we reuse `WORK_WITH_TESTS` permission instead?
It looks very close to the new one by its meaning, but includes wider range of allowable actions for the same entity (test item).